espnet/nets/pytorch_backend/e2e_asr_conformer.py

```diff
@@ -0,0 +1,601 @@
+# Copyright 2020 Johns Hopkins University (Shinji Watanabe)
```

Comment:
Maybe we can override the Transformer ASR class because the only difference should be the encoder. What do you think about? @ShigekiKarita @sw005320

Reply:
I also feel that override would be a better solution.
I agree that the code is still redundant and it's difficult to maintain it.

@pengchengguo, could you try that?
Sorry to ask many things, but this makes your code more useful ant many people would have the benefits.

Reply:
I agree. I will do this.